lib/ci/container.rb

```diff
@@ -1,5 +1,9 @@
 require 'docker'
 
+if Gem::Version.new(Docker::API_VERSION) < Gem::Version.new(1.24)
```

Comment:
Actually. put that in a docker.rb along with the `require 'docker'` then require_relative